src/cpp/src/tokenizer.cpp

```diff
@@ -249,6 +255,11 @@ class Tokenizer::TokenizerImpl {
             if (m_eos_token_id == -1 && content == m_eos_token)
                 m_eos_token_id = std::stoi(key);
         }
+
+        // if pad_token_id not found use eos_token_id as pad_token_id
```

Comment:
please, create a ticket for OV tokenizers and insert TODO to update / drop this code once the ticket is fixed

Reply:
Ticket created

Reply:
CVS-144174